sycl/doc/extensions/experimental/sycl_ext_oneapi_in_order_queue_events.asciidoc

```diff
@@ -116,7 +116,13 @@ a|
 event ext_oneapi_get_last_event() const;
 ----
 | Returns an event representing the execution of the last command submitted to
-the queue.
+the queue. If a call to `ext_oneapi_set_external_event()` on the queue happened
+after all previously submitted commands to the queue, the event passed as the
+argument to that function will be returned.
+
+If the last event was set by a call to `ext_oneapi_set_external_event()`, the
+returned event is guaranteed to compare equal to the event argument of the 
+`ext_oneapi_set_external_event()` call.
```

Comment:
Isn't this sentence just repeating the previous one?

Reply:
I'd reword the first sentence to say:

> If a call to `ext_oneapi_set_external_event()` on the queue happened after all previously submitted commands to the queue, this function returns a copy of the event that was passed to `ext_oneapi_set_external_event()`.

And then delete the second sentence.

Reply:
Much better, thanks!